pyhelper_utils/utils.py

```diff
@@ -0,0 +1,67 @@
+import subprocess
+
+from simple_logger.logger import get_logger
+
+
+LOGGER = get_logger(name=__name__)
+
+TIMEOUT_30MIN = 30 * 60
+
+
+def run_command(
+    command,
+    verify_stderr=True,
+    shell=False,
+    timeout=None,
+    capture_output=True,
+    check=True,
+    hide_log_command=False,
+    **kwargs,
+):
+    """
+    Run command locally.
+
+    Args:
+        command (list): Command to run
+        verify_stderr (bool, default True): Check command stderr
+        shell (bool, default False): run subprocess with shell toggle
+        timeout (int, optional): Command wait timeout
+        capture_output (bool, default False): Capture command output
+        check (bool, default True):  If check is True and the exit code was non-zero, it raises a
+            CalledProcessError
+        hide_log_command (bool, default False): If hide_log_command is True and check will be set to False,
+            CalledProcessError will not get raise and command will not be printed.
+
+    Returns:
+        tuple: True, out if command succeeded, False, err otherwise.
+    """
```

Comment:
Add raises to docstring, when check is `True`, `run` will raise `CalledProcessError` on error.

Reply:
Done, but this part is already mentioned in the doc string, under `check`